packages/api/src/jobs/certification/pdf.js

```diff
@@ -16,18 +16,28 @@ export default (nextQueue, emailQueue) => async (job) => {
     headless: true,
     ignoreDefaultArgs: ['--disable-extensions'],
   });
-  const submission = await rp({
-    uri: `${surveyStackURL}/submissions/${job.data.submission}`,
+
+  const [submission] = await rp({
+    uri: `${surveyStackURL}/submissions?survey=${certifier.survey_id}&match={"_id":{"$oid":"${job.data.submission}"}}`,
     json: true,
+    headers: { Authorization: `${process.env.SURVEY_USER} ${process.env.SURVEY_TOKEN}` },
   });
+
   const survey = await rp({
-    uri: `${surveyStackURL}/surveys/${submission.meta.survey.id}`,
+    uri: `${surveyStackURL}/surveys/${certifier.survey_id}`,
     json: true,
+    headers: { Authorization: `${process.env.SURVEY_USER} ${process.env.SURVEY_TOKEN}` },
   });
+
   if (!submission || !survey) {
     emailQueue.add({ fail: true });
     return Promise.resolve();
   }
+
+  if (!Object.keys(submission.data).length) {
+    console.error(`No data was returned from submission ${job.data.submission}`);
```

Comment:
@kathyavini 

So it is ok to continue on with the export if no survey submission data is returned?

Reply:
Definitely. There are several documents that get exported; the survey is only one of them.

This error is only to help us troubleshoot (e.g. by looking at the logs of the export server) when the issue was accessing survey data in particular.